### PR TITLE
coreutils-full: fix cross build

### PR DESCRIPTION
**Pull request by volth**
###### Motivation for this change

Fix cross build broken in https://github.com/NixOS/nixpkgs/pull/45720

### CODE REVIEW

**Comment by volth** [comment]
@GrahamcOfBorg build pkgsCross.scaleway-c1.coreutils-full

**Comment by volth** [comment]
@GrahamcOfBorg build pkgsCross.armv7l-hf-multiplatform.coreutils-full

File: pkgs/tools/misc/coreutils/default.nix [hunk 1]
```diff
@@ -89,7 +89,7 @@ stdenv.mkDerivation rec {
 
   postInstall = optionalString (stdenv.hostPlatform != stdenv.buildPlatform && !minimal) ''
     rm $out/share/man/man1/*
-    cp ${buildPackages.coreutils}/share/man/man1/* $out/share/man/man1
+    cp ${if minimal then buildPackages.coreutils else buildPackages.coreutils-full}/share/man/man1/* $out/share/man/man1
```
Comment (matthewbauer) [inline pkgs/tools/misc/coreutils/default.nix]: We already have !minimal above - can probably just use coreutils-full here

**Comment by volth** [comment]
@GrahamcOfBorg build pkgsCross.armv7l-hf-multiplatform.coreutils-full

**Comment by Mic92** [comment]
This needs a backport right?

**Comment by volth** [comment]
Maybe.
It is only for cross-compilation whole system closures which is broken on 18.09 in many places (this and cross-compilation ```systemd``` because of recent regressions, but some other (perlPackages) have never worked), so it might have no sense to backport this fix alone

**Comment by Mic92** [comment]
Ok. Then maybe once the base is stable, somebody could create a pull request containing all fixes.

**Pull request merged**